src/D2L.CodeStyle.Analysis/MutabilityInspector.cs

```diff
@@ -64,36 +64,39 @@ public sealed class MutabilityInspector {
 			MutabilityInspectionFlags flags,
 			HashSet<ITypeSymbol> typeStack

```

Comment:
The comment implies this is sane: http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp/Symbols/TypeSymbol.cs,295